index.html

```diff
@@ -235,9 +240,13 @@
             enemies.push(Crafty.e('Enemy').attr({x: 565, y: 400}))
             enemies.push(Crafty.e('Enemy').attr({x: 200, y: 565}))
 
+            for (var i = 0; i < enemies.length; i++) {
+                enemies[i].velocity().x = 1;
```

Comment:
this is a total hack (a sketchy thing to do) because the enemies have to be moving to have their behavior work